exercises/rational-numbers/RationalNumbersTest.cs

```diff
@@ -0,0 +1,222 @@
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
+
+public class RationalNumbersTest
+{
+    [Fact]
+    public void Add_two_positive_rational_numbers()
+    {
+        Assert.Equal(new[] { 7, 6 }, RationalNumbers.Add(new[] { 1, 2 }, new[] { 2, 3 }));
```

Comment:
I think I would prefer to represent a rational number as a `struct`, instead of representing it as an array. This will make the generator a bit more complex, but does make the exercise better I think. However, you can "borrow" from the existing complex numbers implementation, see https://github.com/exercism/csharp/blob/master/generators/Exercises/ComplexNumbers.cs and https://github.com/exercism/csharp/blob/master/exercises/complex-numbers/ComplexNumbersTest.cs

Reply:
OK. I'll do the fix. Actually I wanted to make it like that but then I decided that "excessive creativity" could be not the way to go. AFAISN I wasn't quite right.